build.gradle

```diff
@@ -46,3 +46,10 @@ artifactory {
 task distributeBuild(type: DistributeTask) {
     dependsOn ':artifactoryPublish', 'dexmaker:artifactoryPublish', 'dexmaker-mockito:artifactoryPublish'
 }
+
+repositories {
+    maven {
+        url 'https://maven.google.com/'
```

Comment:
Can you try upgrading to Gradle 4.1 or higher so we can just use `google()` here instead of the `maven` block? If there's any problems with the newer Gradle then just let me know and you can ignore this comment and I'll work on the upgrade later.

Reply:
I also had to update the com.android.tools.build:gradle to 2.3.3, but I think this should not cause issues.